plugin/supervisor.go

```diff
@@ -60,6 +60,7 @@ func newSupervisor(pluginInfo *model.BundleInfo, parentLogger *mlog.Logger, apiI
 		HandshakeConfig: handshake,
 		Plugins:         pluginMap,
 		Cmd:             cmd,
+		Stderr:          wrappedLogger.With(mlog.String("source", "plugin_stderr_panic")).StdErrPanicLogWriter(),
```

Comment:
If I am understanding this correctly. This is the raw stdout of the sub-process so this is likely to be only panics? Do we even need to filter?

Reply:
This would be the ideal solution, given the stream limitation above.

Reply:
Totally agree. I'm just filtering `os.Stderr.WriteString("Message")` kind of logs from the plugin. I'm 0/5 whether we should filter or not.

Reply:
I don't think we need to filter that situation.